gulpfile.js

```diff
@@ -75,14 +79,32 @@ gulp.task('browserify', function() {
     .pipe(gulp.dest(paths.distPublic));
 });
 
-gulp.task('build', ['build-queries', 'copy-js', 'transform-jsx', 'browserify']);
+gulp.task('less', function () {
+  var combined = combiner.obj([
+    gulp.src(paths.lessBundle),
+    less({
+      paths: [
+        'node_modules/bootstrap/less'
+      ]
+    }),
+    gulp.dest(paths.distPublic + '/')
+  ]);
+
+  // any errors in the above streams will get caught
+  // by this listener, instead of being thrown:
+  combined.on('error', console.error.bind(console));
```

Comment:
Yeah, probably. However, before I introduced the "combined" thing they would just fail silently (aka, the build step would succeed with no error message but the actual task would not have been completed, very confusing)... more work is required to figure out how to cause the task to fail in that case...